tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+
+@pytest.fixture()
+def disable_mido_checks(monkeypatch):
+    """
+    Disable internal checks in `mido`, for performance.
+    """
+    from mido.messages import messages
+
+    monkeypatch.setattr(messages, "check_msgdict", lambda d: d)
+
+
+@pytest.fixture()
+def disable_mido_merge_tracks(monkeypatch):
+    """
+    Disallow `mido` from creating `merged_tracks` when files are loaded.
+
+    We don't need `merged_tracks` in our tests, and it's slow to create.
+
+    See https://github.com/mido/mido/pull/565.
```

Comment:
```suggestion
    See https://github.com/mido/mido/pull/565.
    TODO remove these fixtures when PR above is merged and the next Mido version released
```

Reply:
The other fixture is still required after that – `check_msgdict` is very slow too.